PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -296,23 +301,30 @@ local function CreateCamera()
 	end
 
 	function this:ApplyVRPosition()
+		if not HasVRAPI then
+			return
+		end
 		--we only want this to happen in first person VR
 		local player = game.Players.LocalPlayer
-		if not player or not player.Character
-				or not player.Character:FindFirstChild("HumanoidRootPart")
+		if not player or not player.Character 
+				or not player.Character:FindFirstChild("HumanoidRootPart") 
 				or not player.Character.HumanoidRootPart:FindFirstChild("RootJoint") then
 			return
 		end
-
-		if not this:IsInFirstPerson() then
-			local rootJoint = player.Character.HumanoidRootPart.RootJoint
+		
+		local camera = workspace.CurrentCamera
+		local cameraSubject = camera.CameraSubject
+		local isInVehicle = cameraSubject and cameraSubject:IsA('VehicleSeat')		
+		
+		if not UserInputService.VREnabled or not this:IsInFirstPerson() or isInVehicle then
```

Comment:
This line is very confusing to me, are we doing some logic in this function if `VREnabled` is false?